examples/bq-email-exports/terraform/payload.txt

```diff
@@ -0,0 +1,12 @@
+{
+    "project_id": "bq-email-exports",
+    "dataset_id": "bq_exports",
+    "table_name": "bq_export_",
+    "bucket_name": "bq_email_exports",
+    "query_bucket": "query_bucket",
+    "query_file_name": "query.txt",
+    "signed_url_expiration_hrs": 24,
+    "from_email": "sender@example.com",
+    "to_email": "recipient@example.com",
+    "email_subject": "Daily BQ export"
```

Comment:
You should support all things that BigQuery extract supports in this config.
I'd suggest you just a have a nested json block here you specify the API representation of 
[JobConfigurationExtract](https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#JobConfigurationExtract)
Namely the following could all be useful for this use case:
- compression type
- destination format (while I think JSON is a reasonable default the user should be able to control this per their needs.)
- use avro logical types
- field delimeter
https://googleapis.dev/python/bigquery/latest/generated/google.cloud.bigquery.job.ExtractJobConfig.html#google.cloud.bigquery.job.ExtractJobConfig